draft-ietf-quic-transport.md

```diff
@@ -2110,19 +2110,10 @@ infrequently.
 
 Endpoints that use connection IDs with length greater than zero could have their
```

Comment:
This is not part of your PR, but suggested change: "Endpoints that supply non-zero-length connection IDs ..."

Reply:
In reading this, I realized that we are assuming that this assumes that keeping the connection ID is a permissible option.  It isn't.  I'll file a follow-up.